src/components/AddMovie/index.ts

```diff
@@ -0,0 +1,2 @@
+import { AddMovie } from "./AddMovie";
+export default AddMovie;
```

Comment:
You can write it a shorter way:

export { default } from "./AddMovie";

You can apply it here and in all other places.